uninstall.php

```diff
@@ -1,44 +1,24 @@
 <?php
 
 if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
-	return;
+	exit;
 }
 
 if ( ! current_user_can( 'activate_plugins' ) ) {
-	return;
+	exit;
 }
 
-if ( ! defined( 'HMBKP_REQUIRED_WP_VERSION' ) ) {
-	define( 'HMBKP_REQUIRED_WP_VERSION', '3.8.4' );
-}
-
-// Don't activate on old versions of WordPress
-global $wp_version;
-
-if ( version_compare( $wp_version, HMBKP_REQUIRED_WP_VERSION, '<' ) ) {
-	return;
-}
+global $wpdb;
 
-if ( ! defined( 'HMBKP_PLUGIN_PATH' ) ) {
-	define( 'HMBKP_PLUGIN_PATH', plugin_dir_path( __FILE__ ) );
-}
-
-// Load the schedules
-require_once( HMBKP_PLUGIN_PATH . 'hm-backup/hm-backup.php' );
-require_once( HMBKP_PLUGIN_PATH . 'classes/class-services.php' );
-require_once( HMBKP_PLUGIN_PATH . 'classes/class-hmbkp-path.php' );
-require_once( HMBKP_PLUGIN_PATH . 'classes/class-schedule.php' );
-require_once( HMBKP_PLUGIN_PATH . 'classes/class-schedules.php' );
-require_once( HMBKP_PLUGIN_PATH . 'functions/core.php' );
+// Get all schedule options with a SELECT query and delete them.
+$schedules = $wpdb->get_col( $wpdb->prepare( "SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s", 'hmbkp_schedule_%' ) );
 
-$schedules = HMBKP_Schedules::get_instance();
-
-// Cancel all the schedules and delete all the backups
-foreach ( $schedules->get_schedules() as $schedule ) {
-	$schedule->cancel( true );
-}
+array_map( 'delete_option', $schedules );
 
 // Remove the backups directory
+require_once( dirname( __FILE__ ) . '/../hm-backup/hm-backup.php' );
+require_once( dirname( __FILE__ ) . '/class-hmbkp-path.php' );
+require_once( dirname( __FILE__ ) . '/../functions/core.php' );
```

Comment:
And this path should just be ```'/functions/core.php'```. 

But still having issues with the uninstall so still testing.